mlflow/models/__init__.py

```diff
@@ -105,6 +63,7 @@ def build_docker(
 # Under skinny-mlflow requirements, the following packages cannot be imported
 # because of lack of numpy/pandas library, so wrap them with try...except block
 try:
+    from mlflow.models.python_api import predict
```

Comment:
`predict` needs pandas but `build-docker` is not, which needs a bit dirty solution in `predict` impl. Wondering if we really need to make `build-docker` available in mlflow-skinny.

Reply:
We probably don't, but best not to drop it now...thanks for handling this